pkg/utils/release/controller.go

```diff
@@ -39,50 +41,52 @@ func (s *SuiteController) CreateApplicationSnapshot(name string, namespace strin
 }
 
 // CreateRelease creates a new Release using the given parameters.
-func (s *SuiteController) CreateRelease(name, namespace, snapshot, sourceReleaseLink string) (*v1alpha1.Release, error) {
-	release := &v1alpha1.Release{
+func (s *SuiteController) CreateRelease(name, devNamespace, snapshot, sourceReleasePlan string) (*appstudiov1alpha1.Release, error) {
```

Comment:
I still think the parameter name should be `namespace` like it was. This function just creates a Release where we tell it to. What if one day we want to create a release in the managed workspace? There is no benefit to calling the function parameter like this IMO

Reply:
Fixed